release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -147,6 +147,13 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-nw-customize-ingress-error-pages"]
+==== Customizing HAProxy error code response pages
+
+You can specify a custom error code response page for either 503, 404, or both error pages.
```

Comment:
You can specify a custom **http** error code response page for either 503, 404, or both error pages.